index.js

```diff
@@ -251,6 +274,7 @@ function handleResponses(expressApp, options = { swaggerUiServePath: 'api-docs',
   const { specOutputPath, writeIntervalMs } = options;
 
   updateDefinitionsSpec(options.mongooseModels);
+  updateTagsSpec(options.tags);
```

Comment:
So if we write something like `updateTagsSpec(options.mongooseModels || options.tags);` then grouping will be done by mongoose models and no extra tags will be needed. And it still will be two independant features.

Reply:
actually it is better to do it the other way around: `updateTagsSpec(options.tags || options.mongooseModels);` - that will give more controll on grouping

Reply:
Perfect, all set!

- Defaults tags argument to undefined to make `options.tags || options.mongooseModels` work
- Updated `server_basic.js` example to send both `mongooseModels` and `tags`
- Updated `server_advanced.js` example to send only `mongooseModels` (tags taking default value)
- Adjusted README.md with optional fields + default

Let me know 😄